README.md

```diff
@@ -18,17 +18,28 @@ yarn add -D \
     prettier
 ```
 
-In your `eslint.config.cjs`:
+In your `eslint.config.mjs`:
```

Comment:
Based on many past attempts, my opinion is that the (nodejs) world is still not ready for ESM. I think suggesting folks use it invites them to experience more pain than necessary 😅

Reply:
Yeah that has been my experience too for anything that gets imported. 

I only changed this example code because that was the default example code generated by eslint, but doesn't make any diff to me